Tests/Services/AWSRoute53Tests/ListResourceRecordSetsRequestTest.swift

```diff
@@ -1,65 +0,0 @@
-// Code generated by smithy-swift-codegen. DO NOT EDIT!
-
-@testable import AWSRoute53
```

Comment:
do we have a backlog to add it back?

Reply:
Added one here https://github.com/awslabs/aws-sdk-swift/issues/888